provisioning/baremetal_pod.go

```diff
@@ -1065,7 +1065,6 @@ func DeleteMetal3Deployment(info *ProvisioningInfo) error {
 func getPodHostIP(podClient coreclientv1.PodsGetter, targetNamespace string) (string, error) {
 	listOptions := metav1.ListOptions{
 		LabelSelector: metal3AppName,
-		FieldSelector: "status.hostIP",
```

Comment:
I removed this because it failed with;
```
 invalid selector: 'status.hostIP'; can't understand 'status.hostIP'" "name"="provisioning-configuration"
```

Maybe it is because of lack of `get` permission but I did this change for a quick fix

Reply:
As an additional safety measure, can you please also verify that `len(podList.Items) == 1` instead of `> 0`?

Reply:
What do you think for the latest version @andfasano ?

Reply:
Thanks, it seems ok, let's see the virtualmedia job